src/app/pages/DrawerList/page.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+import { Box } from "@mui/system";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
+import InboxIcon from "@mui/icons-material/MoveToInbox";
+import MailIcon from "@mui/icons-material/Mail";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import Link from "next/link";
+import Divider from "@mui/material/Divider";
```

Comment:
```suggestion
import {ListItemIcon, ListItemText, InboxIcon, MailIcon … } from "@mui/material";
```

It is way cleaner than having a huge number of lines.

Reply:
Done..